src/CRUD.php

```diff
@@ -266,6 +268,17 @@ protected function getExecutor(Generic $action)
             return $this->factory($action->ui['executor']);
         }
 
+        // prioritize CRUD addFields over action->fields for Model add action.
+        if ($action->short_name === 'add' && $this->addFields) {
+            $action->fields = $this->addFields;
+        }
+
+        // prioritize CRUD editFields over action->fields for Model edit action.
+        if ($action->short_name === 'edit' && $this->editFields) {
```

Comment:
EditFields should be used in case of addField are not defined for add.

Also here and above the condition should check for null as fields may be an empty array.

Reply:
EditFields and addFields are just a quick way to setup add et edit action fields in CRUD seed, without having to modify the Model action itself, so they should be separated.

Having empty array here is fine since it will rely on $action->fields property instead. 
Remember, this is for specific add and edit Model action so passing an empty array would result in UserAction executor "field" step to directly save record without user input.